ppr-ui/src/views/newMhrRegistration/HomeOwners.vue

```diff
@@ -361,6 +378,13 @@ export default defineComponent({
       disableAddHomeOwnerBtn: computed(
         () => localState.showAddPersonOrganizationSection || localState.showAddPersonSection
       ),
+      // capture different errors in the table to turn off Add Owner buttons and show error
+      hasHomeOwnersTableErrors: computed(
+        () => {
+          return isTransferToExecutorProbateWill.value ? !hasDeletedOwnersWithProbateGrant() : false
+        }
+      ),
```

Comment:
The name seems pretty generic in comparison the very specific error message were showing, perhaps it might help to rename the computed value to be more specific in this case, if the error is for this specific flow?

Reply:
Yea, I wanted to keep the name generic as more error scenarios coming up (AFFE transfer) that we can simply add conditions here to display that error message.